### PR TITLE
examples: fix spdk

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -79,10 +79,12 @@ jobs:
 
       - name: Run tests
         run: |
-          echo wait 5s for SPDK to start... && sleep 5s
-          docker-compose logs
+          docker-compose logs spdk
+          echo wait 55s for SPDK to start... && sleep 55s
```
Comment (mestery) [inline .github/workflows/docker-publish.yml:83]: 55 seconds (!!!!)

**Review by mestery — APPROVED** [review]
Assuming CI passes this LGTM.

File: .github/workflows/docker-publish.yml (hunk 1 above)
Comment (glimchb) [reply .github/workflows/docker-publish.yml:83]: just for debug to see why SPDK fails...

**Comment by mestery** [comment]
> @mestery I don;t understand what SPDK complains about https://github.com/opiproject/opi-prov-life/runs/6993226320?check_suite_focus=true ?
> 
> ```
> Attaching to telegraf_spdk_1
> spdk_1      | [2022-06-21 21:29:07.615384] Starting SPDK v22.05 git sha1 797ff94 / DPDK 22.03.0 initialization...
> spdk_1      | [2022-06-21 21:29:07.616781] [ DPDK EAL parameters: [2022-06-21 21:29:07.616801] spdk_tgt [2022-06-21 21:29:07.616816] --no-shconf [2022-06-21 21:29:07.616830] -c 0x1 [2022-06-21 21:29:07.616843] -m 1024 [2022-06-21 21:29:07.616856] --huge-unlink [2022-06-21 21:29:07.616869] --log-level=lib.eal:6 [2022-06-21 21:29:07.616884] --log-level=lib.cryptodev:5 [2022-06-21 21:29:07.616896] --log-level=user1:6 [2022-06-21 21:29:07.616910] --iova-mode=pa [2022-06-21 21:29:07.616923] --base-virtaddr=0x200000000000 [2022-06-21 21:29:07.616935] --match-allocations [2022-06-21 21:29:07.616949] --file-prefix=spdk_pid15 [2022-06-21 21:29:07.616963] ]
> spdk_1      | TELEMETRY: No legacy callbacks, legacy socket not created
> spdk_1      | [2022-06-21 21:29:08.221356] app.c: 603:spdk_app_start: *NOTICE*: Total cores available: 1
> spdk_1      | [2022-06-21 21:29:08.423947] reactor.c: 947:reactor_run: *NOTICE*: Reactor started on core 0
> spdk_1      | [2022-06-21 21:29:08.424105] accel_engine.c: 969:sw_accel_engine_init: *NOTICE*: Accel framework software engine initialized.
> ```
> 
> why unhealthy ?
> 
> ```
>        Name                      Command                   State                         Ports                  
> ----------------------------------------------------------------------------------------------------------------
> telegraf_grafana_1    /run.sh                          Up (healthy)     0.0.0.0:3000->3000/tcp,:::3000->3000/tcp
> telegraf_influxdb_1   /entrypoint.sh influxd           Up (healthy)     0.0.0.0:8086->8086/tcp,:::8086->8086/tcp
> telegraf_spdk_1       sh -c echo 2048 > /proc/sy ...   Up (unhealthy)   0.0.0.0:9009->9009/tcp,:::9009->9009/tcp
> telegraf_telegraf_1   /entrypoint.sh telegraf          Up (healthy)     8092/udp, 8094/tcp, 8125/udp
> ```

Can you replicate the failure locally or it only fails in CI? I will try this on my own and see what is going on.

**Comment by glimchb** [comment]
locally all works well

**Pull request merged**